### ISSUE TITLE
Add a Go To Link for the website form field

### ISSUE THREAD

**Issue by SAUMILDHANKAR** [hackforla/food-oasis#2273]
### Overview
We would like to have a Go To Link pop up once the user starts entering a website URL in contact details. This would be helpful for validation.

### Action Items
- [ ] Create a functionality to add a Go To Link when user enters any character in website field.
- [ ] Test in the dev environment to make sure the link redirects correctly.
- [ ] Once the PR associated with this issue is merged or this issue is closed, check issue #2128 and mark the corresponding action item complete. Further, if all the action items in that issue are marked complete, close that issue as well.

### Resources/Instructions
- Related link: https://devla.foodoasis.net/admin/organizationedit/
- There is similar functionality when we add text in instagram field which was added with PR #2203 
<img width="894" alt="issue 2273" src="https://github.com/user-attachments/assets/71d260ea-c465-4cf5-a51d-9c5a3c9fa973">

**Comment by SAUMILDHANKAR** [comment]
@JmJenna Reached out on slack to check your availability. Please let us know when your availability changes and we can find another issue for you to work on.
